### DISCUSSION TITLE
cookies are not vaild

### DISCUSSION THREAD

**Discussion by MTomaszek** [kevinzg/facebook-scraper#514] [Q&A] [answered]
I try to run simple example, but got msg that I need to be  logged in.  so I  set cookies param, then I got
....local/lib/python3.7/site-packages/facebook_scraper/__init__.py", line 36, in set_cookies
    raise exceptions.InvalidCookies(f"Cookies are not valid")
facebook_scraper.exceptions.InvalidCookies: Cookies are not valid

I pasted export from Cookie Quick Manager.
My program is simple 
```
from facebook_scraper import get_posts

for post in get_posts('nintendo', pages=1,  cookies=('cookies.txt')):
    print(post['text'][:50])

``` 
how can I point my proxy to see traffic or extend logging to see more details what is going on. 
Also what should I do to use cli ?

**Comment by neon-ninja** [comment, marked as answer]
After extracting cookies, make sure you don't click the logout button, as that would invalidate your cookies
You can use `set_proxy` to set a proxy. You can use `enable_logging(logging.DEBUG)` to output more logs. You can call `facebook-scraper` from CLI.

**Reply by MTomaszek** [reply]
In FB I'm still logged in,  I have noticed that some characters of content raw  for xs cookie are encoded, I assume it's limitation of json file. maybe it should  work with these character, but could help me how to provide cookies in flat txt file, I saw that code there are some possibilities, but I don't have idea how to format it.

**Reply by neon-ninja** [reply]
In the settings for Cookie Quick Manager, you can set the output format to netscape format
